src/main/resources/static/views/create-container.html

```diff
@@ -115,17 +115,14 @@
 
             </div>
 
-            <div class="checkbox space-before">
-                <label>
-                    <input type="checkbox" ng-model="ackEula">
-                    Accept the <a href="https://account.mojang.com/documents/minecraft_eula" target="_blank">Minecraft
-                    EULA</a>
-                </label>
+            <div class="btn-group" role="group" aria-label="Eula buttons">
```

Comment:
To avoid the outer `btn-group` hanging around, see if it works the same to put

    ng-show="!ackEula"

on that element instead of two inner ones?